geopandas/base.py

```diff
@@ -2574,6 +2574,118 @@ def hausdorff_distance(self, other, align=True, densify=None):
         """
         return _binary_op("hausdorff_distance", self, other, align, densify=densify)
 
+    def frechet_distance(self, other, align=True, densify=None):
+        """Returns a ``Series`` containing the Frechet distance to aligned `other`.
+
+        The Frechet distance is the largest distance consisting of any point in `self`
+        with the nearest point in `other`.
+
+        The operation works on a 1-to-1 row-wise manner:
+
+        .. image:: ../../../_static/binary_op-01.svg
+           :align: center
+
+        Parameters
+        ----------
+        other : GeoSeries or geometric object
+            The Geoseries (elementwise) or geometric object to find the
+            distance to.
+        align : bool (default True)
+            If True, automatically aligns GeoSeries based on their indices.
+            If False, the order of elements is preserved.
+        densify : float (default None)
```

Comment:
I think that in the general description above is missing the information that the algorithm [considers only vertices](https://shapely.readthedocs.io/en/stable/reference/shapely.frechet_distance.html#shapely.frechet_distance). Without that, it is unclear what is the benefit of `densify` keyword.

Reply:
fixed in docstring, specified vertex in general description